sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -88,12 +91,26 @@ export function record(
     testTitle = testContext.test!.title;
   }
 
+  const stringTest = testContext.currentTest!.fn!.toString();
+  const currentHash = MD5(stringTest);
+  const testAbsolutePath = testContext.currentTest!.file!;
+
+  if (
+    isSoftRecordMode() &&
+    !testHasChanged(testHierarchy, testTitle, testAbsolutePath, currentHash)
+  ) {
+    testContext.test!.title = `${
+      testContext.test!.title
+    } (The test has been skipped since it hasn't changed)`;
```

Comment:
maybe something like `Test unchanged since last recording` ?

Reply:
I like that!